package.json

```diff
@@ -9,6 +9,7 @@
   "main": "src/index.js",
   "license": "MIT",
   "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
```suggestion
```

Reply:
> @eSlider, this is a nice PR, why did you close it ?

@lovasoa, it's breaks previous behavior.
 
Here are the changes:

* [x] Download an sqlite database file (axios)
* [x] Prepare downloaded file to handle it right (that was not easy :)
* [x] Add favicon to prevent 404 errors 
* [x] Remove load of `manifest.json` - that not exists
* [x] Lock packages to get working version
* [x] Add an select SQL query as a value for textearea  
* [x] Publish to another place: [link](https://sqlite-js-demo.netlify.app/)
* [x] Circle CI: 
  * [x] Automated static content build 
  * [x] Publish build in a `gh-pages` branch
  * [x] Disable [jekyll](https://jekyllrb.com/) post processing
  * [x] Prepare [demo](https://produktor.github.io/react-sqljs-demo/) to work over github pages CDN 
  * [ ] Get load generated assets using relative paths  (That what I am working on)   

If you ok with, I will open PR again. 

P.S. And thank you so much for you simple react-boilerplate! It's amazing!

Reply:
Hello !

>  Download an sqlite database file (axios)

You don't need to add a dependency just to make an HTTP request. Let's just use `fetch`

>  Prepare downloaded file to handle it right (that was not easy :)

?

>  Add favicon to prevent 404 errors

:heavy_check_mark: 

>  Lock packages to get working version

:heavy_check_mark: 

> Add an select SQL query as a value for textearea

:heavy_check_mark: 

>  Publish to another place: link
    Circle CI:
    Automated static content build
    Publish build in a gh-pages branch
    Disable jekyll post processing
    Prepare demo to work over github pages CDN
    Get load generated assets using relative paths (That what I am working on)

Let's just keep the current netlify CD (https://sqljs-react-demo.netlify.com/). I don't think we need to switch to github pages.